PSKoans/Koans/Cmdlets 1/AboutPSProviders.Koans.ps1

```diff
@@ -127,17 +127,19 @@ Describe 'Environment Provider' {
 }
 
 Describe 'FileSystem Provider' {
-    $Path = 'TestDrive:' | Join-Path -ChildPath 'File001.tmp'
+    #Get-TempDirectory is a private function in PSKoans
+    #This should return a path to your temp directory.
+    $Path = Get-TempDirectory
```

Comment:
Private functions aren't visible to the koans during execution. It'll need to be a public function, unfortunately. 🤔 Pester has TestDrive, I wonder if they expose any functions for this... Hm, doesn't look like it unfortunately.  Except for `TestDrive:\` -- might be a simpler alternative given it's an already-available solution, perhaps?

I'm not sure.

/cc @nohwnd is `TestDrive:` reliable on unix systems?

Reply:
Yes :) it should work at least as well as on windows.